### PR TITLE
added automation tag and automation task

### PR DESCRIPTION
**Pull request by VardhanDeva**

### CODE REVIEW

**Review by sathishkumar-p — APPROVED** [review]

File: packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml [hunk 1]
```diff
@@ -80,36 +80,41 @@
...
     creates: "{{dss_home_directory}}"
     remote_src: yes
-  tags: [dss-upgrade-production, dss-upgrade-training]
+  tags: [ dss-upgrade-production, dss-upgrade-training, dss-upgrade-automation ]
 
 - name: Run DSS upgrade
   become_user: "{{dss_system_user}}"
   shell: yes Y | {{dss_home_directory}}/installer.sh -d {{dss_data_directory}} -u
-  tags: [dss-upgrade-production, dss-upgrade-training]
+  tags: [ dss-upgrade-production, dss-upgrade-training, dss-upgrade-automation ]
```
Comment (sathishkumar-p) [inline packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml:88]: remove the tag dss-upgrade-automation

File: packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml [hunk 2]
```diff
@@ -80,36 +80,41 @@
...
+
+- name: Run DSS upgrade
+  become_user: "{{dss_system_user}}"
+  shell: yes Y | {{dss_home_directory}}/installer.sh -d {{dss_data_directory}} -t automation -u
+  tags: [dss-upgrade-automation ]
```
Comment (sathishkumar-p) [inline packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml:94]: make the name Run DSS upgrade automation

Comment (VardhanDeva) [reply packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml:94]: Changed the name

File: packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml [hunk 3]
```diff
@@ -80,36 +80,41 @@
...
   shell:
     cmd: |
       /etc/init.d/dataiku start
-  tags: [dss-upgrade-production, dss-upgrade-training]
+  tags: [ dss-upgrade-production, dss-upgrade-training, dss-upgrade-automation ]
 
 #Cleanup
 - name: Remove backup file from local storage
   file:
     path: "{{dss_backup_directory}}/dataiku_backup_{{date_now}}_{{ansible_eth0.ipv4.address}}.tar.gz"
```
[the remark this answers is not in the document]
Comment (VardhanDeva) [reply packages/dataiku/azure/ansible/roles/dataiku-upgrade/tasks/main.yml:118]: Sure , I am currently working on it . If it works I will raise a new PR on Monday .

**Review by JanMigon — APPROVED** [review]

**Pull request merged**